biscuit-parser/src/parser.rs

```diff
@@ -2167,4 +2171,24 @@ mod tests {
             expected_checks
         );
     }*/
+
+
+    #[test]
+    fn chained_calls() {
+        use builder::{int, set, Binary, Op};
+
+        assert_eq!(
+            super::expr("[1].intersection([1]).contains(1)").map(|(i, o)| (i, o.opcodes())),
+            Ok((
+                "",
+                vec![
+                    Op::Value(set([int(1)].into_iter().collect())),
+                    Op::Value(set([int(1)].into_iter().collect())),
+                    Op::Binary(Binary::Intersection),
+                    Op::Value(int(1)),
+                    Op::Binary(Binary::Contains)
```

Comment:
i'd throw in a unary method to make sure parsing both binary and unary methods are correctly handled.

It might be good to add the example to the samples as well.